baremetal/metal3data_manager.go

```diff
@@ -355,13 +366,16 @@ func (m *DataManager) getAddressesFromPool(ctx context.Context,
 			// network.IPAddressFromIPPool
 			for _, route := range network.Routes {
 				if route.Gateway.FromIPPool != nil {
+					m.Log.Info("Fetch Route Gateway from IP Pool", "Pool Name", route.Gateway.FromIPPool)
```

Comment:
Please see my previous comment re: unifying usage of IPPool here in all other occurences